flybywiresim/aircraft
# chore(a380): Adds TCAS to the A380X

Fixes #8724 

## Summary of Changes

1. Edits the package.json so the TCAS system will build for the A380X
2. Updates the fbw-a380x/src/systems/tcas/rollup.config.js file to have the correct output. 

## Screenshots (if necessary)

## References


## Additional context
Adjusted the Tcas code so it will build in the a380x.

Discord username (if different from GitHub):

## Testing instructions

- Load into the A380X in the simulator
- Use the traffic injector to inject traffic
- Confirm that the aircraft's TCAS recognizes the traffic and adjusts accordingly

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
